Makefile

```diff
@@ -649,7 +652,7 @@ NPROC_POOL ?= 1
 export NPROC_POOL
 
 run-% : $(OBJDIR)/%
-	@$(PYTHON) tests/junit.py --mode tap --ceed-backends $(BACKENDS) --nproc $(NPROC_TEST) --pool-size $(NPROC_POOL) $(<:$(OBJDIR)/%=%)
+	@$(PYTHON) tests/junit.py --mode tap --ceed-backends $(BACKENDS) $(if $(or $(PETSC_ARCH),$(PETSC_DIR)),--petsc-arch $(or $(PETSC_ARCH),$(PETSC_DIR)) )$(if $(SMARTREDIS_DIR), --smartredis_dir $(SMARTREDIS_DIR) )--nproc $(NPROC_TEST) --pool-size $(NPROC_POOL) $(<:$(OBJDIR)/%=%)
```

Comment:
Expanding mentally, it looks like this can produce `--petsc-arch /path/to/installed/petsc`, which is pretty confusing. But if the variable needs to be set, can the script just read the environment variable instead of command-line argument?

Reply:
We intentionally passed the values to the script when we set this up for Ratel so its explicit what triggers the behavior in the script. We could revisit this for both Ratel and here. @zatkins-dev thoughts?